### ISSUE TITLE
Deploy to staging env?

### ISSUE THREAD

**Issue by gloriousjob** [aws/aws-dotnet-deploy#601]
## The Question
Hi, I use an ElasticBeanstalk with .net 6.0.3 and would like to have a Staging configuration deploy instead of the Release configuration (or let's say Debug instead of Staging if that's easier to consider). How can I do this?

**Environment (please complete the following information):**
 - CLI Version: aws.deploy.cli      0.40.18
 - OS Info: Mac OS X

---

This is a :question: general question

**Comment by ashishdhingra** [comment]
@gloriousjob Please confirm if you still need guidance on this issue. If yes, please elaborate your use case.

Thanks,
Ashish

**Comment by gloriousjob** [comment]
No, I think I am good. The problem I had was that this setting was hidden under the advanced settings, which I hadn't noticed.

**Comment by gloriousjob** [comment]
Sorry, I haven't been working on this project as much but I'd like to handle it when this project has issues. I'm not sure if this is a new bug or if I just didn't notice from back when I replied but I tried changing the Dotnet Build Configuration to Staging. 
So basically, it appears that it made a Staging build but when it deployed to Elastic Beanstalk, connections weren't working. I logged into the ec2 and replaced the appsettings.production.json with appsettings.staging.json and restarted the app server. At this point, the connections started working. It seems as though building in Staging doesn't utilize the appsettings.Staging.json file. Have I misconfigured how Staging works or this is a bug from this end?
